resolwe/test/testcases/process.py

```diff
@@ -92,30 +92,52 @@ def _register_schemas(self, **kwargs):
         global variable based on ``path`` key in ``kwargs``.
 
         """
-        Process.objects.all().delete()
+        def remove_pks(schemas):
+            """Remove primary keys from the given schemas."""
+            for s in schemas:
+                s.pk = None
+            return schemas
+
+        schemas_types = [
+            {
+                'name': 'processes',
+                'model': Process,
+            },
+            {
+                'name': 'descriptor_schemas',
+                'model': DescriptorSchema,
+            },
+        ]
+
+        for schemas in schemas_types:
+            schemas['model'].objects.all().delete()
```

Comment:
Hmm... I haven't checked. I just saw that we did:

```python
Process.objects.all().delete()
```

previously and this does the same except that it also deletes all descriptor schemas which I think is more consistent.